iceoryx_posh/source/runtime/posh_runtime.cpp

```diff
@@ -104,10 +104,9 @@ const ProcessName_t& PoshRuntime::verifyInstanceName(cxx::optional<const Process
         LogError() << "Cannot initialize runtime. Application name must not be empty!";
         std::terminate();
     }
-    else if (name.value()->c_str()[0] != '/')
+    else if (name.value()->c_str()[0] == '/')
```

Comment:
Just for completeness, this will introduce an API change to all existing projects because they need to remove the slash now from the process name. @budrus @marthtz @sculpordwarf

Reply:
there are so many API changes, this butters no parsnips